config.json

```diff
@@ -2,6 +2,7 @@
   "language": "Groovy",
   "checklist_issue": 9,
   "active": false,
+  "blurb": "Apache Groovy is an optionally typed, optionally dynamic, object-oriented and functional language which aims to improve developer productivity and happiness.  It is fully compatible with the JVM ecosystem.",
```

Comment:
I propose the following text:

"Groovy is an unopinionated multi-paradigm language that enhances Java with intent to make your code less verbose but still utilizing the power of JVM"

'Invented' it myself, so feel free to correct something

Reply:
I love it!

Reply:
@m-a-ge @kytrinyx Hear, hear!